cmd/node/factory/structsStatusHandlers.go

```diff
@@ -178,6 +180,18 @@ func (shi *statusHandlersInfo) LoadTpsBenchmarkFromStorage(
 	return okTpsBenchmarks
 }
 
+func (shi *statusHandlersInfo) updateTpsMetrics(metricsMap map[string]interface{}) {
+	for key, value := range metricsMap {
+		if key == core.MetricAverageBlockTxCount {
+			log.Trace("setting metric value", "key", key, "value string", value.(string))
+			shi.StatusHandler.SetStringValue(key, value.(string))
```

Comment:
You can add continue here and drop the else branch

Reply:
+1